Documentation/Functions/Typolink.rst

```diff
@@ -428,6 +428,21 @@ returnLast
     So, in these two cases you will not get the value wrapped but the URL
     or target value returned!
 
+    ..  versionadded:: 11.4
+        If set to ``result``, it will return the json_encoded output of the
```

Comment:
Can you remove one indentation for the text (and add an empty line)? We use `.. versionadded:: 11.4` just as a "header". Then it is fine for me.

So like 

    ..  versionadded:: 11.4

    If set to ....

Reply:
@brotkrueml, sure thing. done.

You might wanna check other parts of the docs then, since I got that inline-ness from somewhere in the docs. E.g. in here
* `Documentation/Conditions/Index.rst` and
* `Documentation/ContentObjects/Fluidtemplate/DataProcessing/*.rst`